regression/test.pl

```diff
@@ -59,7 +59,19 @@ ($)
 
 sub test($$$$$) {
   my ($name, $test, $t_level, $cmd, $ign) = @_;
-  my ($level, $input, $options, @results) = load("$test");
+  my ($level, $input, $options, $grep_options, @results) = load("$test");
+
+  # If the 4th line starts with a '-' we use that line as options to pass to
+  # grep when matching all lines in this test
+  if($grep_options =~ /-/) {
+    print "\nActivating perl flags: $grep_options\n";
```

Comment:
You mean the print statement? I mainly added to verify other tests were not inadvertently using this but given that it would be bad for this to be accidentally activated I thought it was generally useful.